CONTRIBUTING.md

```diff
@@ -2,8 +2,8 @@
 
 ### Setup
 ```sh
-$ git clone https://github.com/babel/babili
-$ cd babili
+$ git clone https://github.com/babel/minify
```

Comment:
maybe this should be `git clone https://github.com/babel/minify babel-minify`? and need to change below too